cluster/gce/configure-vm.sh

```diff
@@ -83,10 +83,18 @@ for k,v in yaml.load(sys.stdin).iteritems():
 }
 
 function remove-docker-artifacts() {
+  echo "== Deleting docker0 =="
+  # Forcibly install bridge-utils (options borrowed from Salt logs).
+  until apt-get -q -y -o DPkg::Options::=--force-confold -o DPkg::Options::=--force-confdef install bridge-utils; do
+    echo "== install of bridge-utils failed, retrying =="
+    echo sleep 5
```

Comment:
Hah. Good catch. Probably not, no. Will fix in front of a real computer.
On Mar 29, 2015 2:26 PM, "David Oppenheimer" <notifications@github.com>
wrote:

> In cluster/gce/configure-vm.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6148#discussion_r27359276>
> :
>
> > @@ -83,10 +83,18 @@ for k,v in yaml.load(sys.stdin).iteritems():
> >  }
> >
> >  function remove-docker-artifacts() {
> > +  echo "== Deleting docker0 =="
> > +  # Forcibly install bridge-utils (options borrowed from Salt logs).
> > +  until apt-get -q -y -o DPkg::Options::=--force-confold -o DPkg::Options::=--force-confdef install bridge-utils; do
> > +    echo "== install of bridge-utils failed, retrying =="
> > +    echo sleep 5
>
> did you really mean "echo sleep 5" or just "sleep 5" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6148/files#r27359276>
> .
>